client/lib/application/src/application-lifecycle/components/Error.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const Error = (props) => {
```

Comment:
Maybe reuse the existing error component? https://github.com/zalando-stups/yourturn/blob/master/client/lib/common/src/error.jsx